README.rst

```diff
@@ -119,6 +120,38 @@ You can also easily deserialize JSON-LD to python objects
     #<Book(_id="http://example.com/books/1", name="Ilias")>
 
 
+Annotations
+-----------
+
+Classes can also be annotated directly with schema information, removing the need to have a separate schema. This
+can be done by setting the ``metaclass`` of the model to ``JsonLDAnnotation``.
+
+.. code-block:: python
+
+    import datetime.datetime as dt
+
+    from calamus import JsonLDAnnotation
+    import calamus.fields as fields
+
+    schema = fields.Namespace("http://schema.org/")
+
+    class User(metaclass=JsonLDAnnotation):
+        class Meta:
+            rdf_type = schema.Person
+        _id = fields.Id()
+        birth_date = fields.Date(schema.birthDate, default=dt.now)
+        name = fields.String(schema.name, default=lambda: "John")
```

Comment:
For ease of comparison, I would keep the ordering here the same as for `BookSchema` above:

```suggestion
        _id = fields.Id()
        birth_date = fields.Date(schema.birthDate, default=dt.now)
        name = fields.String(schema.name, default=lambda: "John")
        
        class Meta:
            rdf_type = schema.Person
```